### ISSUE TITLE
[Bug]: Ignore .git directory when using file system mirror

### ISSUE THREAD

**Issue by klein0r** [ioBroker/ioBroker.javascript#1685]
### I'm sure that

- [X] This issue is still present in the **current beta version** of this adapter
- [X] There is no other (open) issue with the same topic (use the search!)
- [X] This issue is not described in the adapter documentation / FAQ (read the docs!)

### Script type

all

### The problem

```
Error while trying to watch folder /home/iobroker/scripts/.git/objects/ff/: Error: ENOENT: no such file or directory, stat '/home/iobroker/scripts/.git/objects/ff/tmp_obj_Ix6Tb4'
```

### Version of nodejs

20.16.0

### Version of ioBroker js-controller

6.0.9

### Version of adapter

8.8.2

**Comment by GermanBluefox** [comment]
Hm... It is a strange error, as all folders and files should be filtered out by this: https://github.com/ioBroker/ioBroker.javascript/blob/master/lib/mirror.js#L63

**Comment by klein0r** [comment]
> as all folders and files should be filtered out by this:

Yes, ... I was not able to reproduce this issue in my dev-server environment.
